uber/templates/preregistration/group_members.html

```diff
@@ -100,9 +103,6 @@
             <td> <a style="font-size:10pt" href="register_group_member?group_id={{ group.id }}">Register someone for this badge</a>: </td>
             <td colspan="3">
                 {{ attendee.badge }}
-                {% if attendee.amount_extra %}
```

Comment:
Hahaha, yes, this looks like it was an error. Those nested if statements can throw a person off.